grl/environment/examples_lib.py

```diff
@@ -626,6 +628,91 @@ def simple_chain(n: int = 10):
 
     return to_dict(T, R, 0.9, p0, phi, Pi_phi)
 
+def tmaze_5_two_thirds_up():
+    # n_obs x n_actions
+    n = 5
+    discount = 0.9
+    Pi_phi = [
+        np.array([[0, 0, 1, 0], [0, 0, 1, 0], [0, 0, 1, 0], [2 / 3, 1 / 3, 0, 0], [1, 0, 0, 0]])
+    ]
+
+    # memory policy is observations * memory bits (2) x n_actions
+    Pi_phi_x = [Pi_phi[0].repeat(2, axis=0)]
+    return to_dict(*tmaze(n, discount=discount), Pi_phi, Pi_phi_x)
+
+def tmaze_5_two_thirds_up_fully_observable():
+    # n_obs x n_actions
+    n = 5
+    discount = 0.9
+    T, R, discount, p0, phi = tmaze(n, discount=discount)
+    phi_fully_observable = np.eye(T.shape[-1])
+
+    pi = np.zeros((T.shape[-1], 4))
+    pi[:, 2] = 1
+    pi[-2, :] = 0
+    pi[-2, 0] = 2 / 3
+    pi[-2, 1] = 1 / 3
+
+    Pi_phi = [pi]
+
+    # memory policy is observations * memory bits (2) x n_actions
+    Pi_phi_x = [Pi_phi[0].repeat(2, axis=0)]
+    return to_dict(T, R, discount, p0, phi_fully_observable, Pi_phi, Pi_phi_x)
+
+def tmaze_5_two_thirds_up_almost_fully_observable():
```

Comment:
Could use a comment explaining what this means

Reply:
I'll just take this out, not really necessary.

Reply:
It's to test out t-maze, except you always know where the reward is, but all corridor observations are aliased. It was mainly to see if in the "optimal 1-bit memory" case for t-maze we still have a lambda discrepancy due to the corridor aliasing (we do).

Reply:
I've added a comment to explain what this is.